lib/rubocop/cop/layout/closing_parenthesis_indentation.rb

```diff
@@ -155,15 +155,19 @@ def expected_column(left_paren, elements)
         end
 
         def all_elements_aligned?(elements)
-          elements.flat_map do |e|
-            if e.hash_type?
-              e.each_child_node.map { |child| child.loc.column }
-            else
+          if all_kwargs?(elements)
+            elements.first.each_child_node.map { |child| child.loc.column }
+          else
+            elements.flat_map do |e|
               e.loc.column
             end
           end.uniq.count == 1
         end
 
+        def all_kwargs?(elements)
+          elements.count == 1 && elements.first.hash_type?
```

Comment:
Is `elements.count == 1 &&` necessary? If so, can you add a corresponding test for it?